src/extension/ncwms/src/main/java/applicationContext.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE beans PUBLIC "-//SPRING//DTD BEAN//EN" "http://www.springframework.org/dtd/spring-beans.dtd">
+
+<beans>
+    <!-- Spring will reference the instance of the HelloWorld class
+           by the id name "helloService" -->
```

Comment:
Haha! Forgot that there! :)

It's from the boilerplate of the plugin...

I'll remove it + the corresponding function in the java class.